frontend-react/src/components/Crumbs.tsx

```diff
@@ -11,28 +16,49 @@ interface CrumbConfig {
 interface CrumbsProps {
     crumbList?: CrumbConfig[];
     noPadding?: boolean;
+    previousPage?: string;
 }
 
-const Crumbs = ({ crumbList, noPadding }: CrumbsProps) => {
-    if (crumbList) {
+const Crumbs = ({ crumbList, noPadding, previousPage }: CrumbsProps) => {
+    if (crumbList || previousPage) {
         return (
             <div className={!noPadding ? "grid-container" : ""}>
-                <BreadcrumbBar>
-                    {crumbList?.map((crumbConfig) => (
-                        <Breadcrumb
-                            key={`${crumbConfig.label}`}
-                            current={!crumbConfig?.path}
+                {previousPage !== undefined ? (
+                    <div className="font-sans-lg line-height-sans-4 margin-top-8">
+                        <IconButton
+                            size="big"
+                            type="button"
+                            onClick={() => window.history.back()}
```

Comment:
Technically that would work, though seems weird since I would be passing the event as an argument to the back function (in this case it would technically work since back doesn't use arguments).

Reply:
Definitely not an issue -- just thinking about [the discussion Kevin and I had yesterday](https://github.com/CDCgov/prime-reportstream/pull/7883#issuecomment-1374998121).  This might be a case in which we could just pass the reference straight through rather than wrapping it in an anonymous function, but not a huge deal either way!

Reply:
Yeah I've always been a bit confused on best practices with event handler anonymous functions when the supposed golden rule is to not use them to prevent memory churn in re-renders. Is it that big of a problem on event handlers? Do we need to pull out hook tooling to memo every single one? etc.